src/QsCompiler/Tests.Compiler/LinkingTests.fs

```diff
@@ -110,11 +112,10 @@ type LinkingTests (output:ITestOutputHelper) =
         compilationDataStructures.Diagnostics() |> Seq.exists (fun d -> d.IsError()) |> Assert.False
 
         Assert.NotNull compilationDataStructures.BuiltCompilation
-        let mutable monomorphicCompilation = compilationDataStructures.BuiltCompilation
-        compilationDataStructures.BuiltCompilation.Monomorphisize &monomorphicCompilation |> Assert.True
+        let monomorphicCompilation = ResolveGenericsSyntax.Apply compilationDataStructures.BuiltCompilation
```

Comment:
Could we rename ResolveGenericsSyntax to something like Monomorphize (I am open to other suggestions)?

Reply:
I added a wrapper class around the transformations called MonomorphizationTransfromation.